src/app/messages/es/messages.json

```diff
@@ -154,10 +154,12 @@
   "i18n:prevnext:prev:aria-label": "Página anterior",
   "i18n:prevnext:prev:text": "Anterior",
   "i18n:search-results:bar": "Barra lateral de resultados de búsqueda",
-  "i18n:search-results:bar:header:title": "Resultados de búsqueda",
+  "i18n:search-results:bar:header:title:plain": "Buscar",
```

Comment:
@bethshook If you get a chance, can you peek at these and let me know if they sound right? 🙏